torture/src/supervisor/pbt.rs

```diff
@@ -0,0 +1,48 @@
+//! Collection of process backtraces.
+//!
+//! This uses the grug-brain developer approach: just invoke the LLDB or GDB to get the backtrace.
+
+use tokio::process::Command;
+use which::which;
+
+pub async fn collect_process_backtrace(pid: u32) -> anyhow::Result<String> {
+    // Determine which debugger tool to use.
+    let command_str = if which("lldb").is_ok() {
+        lldb(pid)
+    } else if which("gdb").is_ok() {
+        gdb(pid)
+    } else {
+        anyhow::bail!("Neither lldb nor gdb are available in PATH.")
```

Comment:
Maybe would be helpful to say _why_ torture is looking for those tools. For example, "attempted to print backtrace of stalled process. ensure lldb or gdb are in PATH"

Reply:
Actually, we print that up the stack!